### PR TITLE
search bar

### PR DESCRIPTION
**Pull request by pvnovikova**
- [DEMO LINK](https://pvnovikova.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://pvnovikova.github.io/layout_search-bar-airbnb/report/html_report/)

[x]Icon implemented using background-image CSS property
[x] Inputs are written inside of 'form' tag with correctly passed attributes
[x]All Typical Mistakes from BEM lesson theory are checked.
[x] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,120 @@
...
+  font-style: normal;
+  src: url(/src/fonts/Avenir-Book.ttf) format('ttf');
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  src: url(/src/fonts/Avenir-Heavy.ttf) format('ttf');
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:18]:
```suggestion
@font-face {
  font-family: Avenir;
  font-weight: 300;
  font-style: normal;
  src: url(/src/fonts/Avenir-Book.ttf) format('ttf');
}

@font-face {
  font-family: Avenir;
  font-weight: 600;
  font-style: normal;
  src: url(/src/fonts/Avenir-Heavy.ttf) format('ttf');
}
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
image
image

**Review by pvnovikova — COMMENTED** [review]
[DEMO LINK](https://pvnovikova.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://pvnovikova.github.io/layout_search-bar-airbnb/report/html_report/)
[x]Icon implemented using background-image CSS property
[x] Inputs are written inside of 'form' tag with correctly passed attributes
[x]All Typical Mistakes from BEM lesson theory are checked.
[x] Code follows all the [Code Style Rules ❗️](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

**Comment by pvnovikova** [comment]
will create new

**Pull request closed without merging**